idl/github.com/uber/cadence/admin.thrift

```diff
@@ -49,3 +59,23 @@ struct DescribeWorkflowExecutionResponse{
   40: optional string mutableStateInCache
   50: optional string mutableStateInDatabase
 }
+
+//At least one of the parameters needs to be provided
+struct DescribeHistoryHostRequest {
```

Comment:
Should we put the request and response struct to shared.thrift, to avoid redundancy?

Reply:
That sounds good to me.